src/task3/data-access/user.js

```diff
@@ -0,0 +1,156 @@
+import uuidv4 from 'uuid/v4';
+import { User } from '../models/User';
+import { db } from '../config/database';
+
+const { Op } = require('sequelize');
+
+function get(req, res) {
+    if (req.query.loginSubstring && req.query.limit) {
+        User.scope('active')
+        .findAll({
+            where: {
+                login: {
+                    [Op.substring]: req.query.loginSubstring
+                }
+            },
+            limit: req.query.limit,
+            order: [[db.col('login'), 'ASC']]
+        })
+        .then((users) => {
+            if (users) {
+                res.status(200).send({
+                    success: 'true',
+                    users
+                });
+            } else {
+                res.status(404).send({
+                    success: 'false',
+                    message: 'Trere are not any person with this login'
+                });
+            }
+        })
```

Comment:
We shouldn't make any request/response manipulation in scope of work with data access layer.
This handlers should be placed in scope `services` layer. I'll prepare an example in scope of you solution